lib/helpers.js

```diff
@@ -42,20 +42,48 @@ const format = (event, options = { ignoreSelection: false }) => {
   const selectedText = editor.getSelectedText();
   const isTransformingFile = options.ignoreSelection || !selectedText;
 
-  const textToTransform = isTransformingFile ? editor.getText() : selectedText;
+  const currentScope = getCurrentScope();
+  const isEmbedded = ['text.html.vue', 'text.html.basic'].includes(currentScope);
```

Comment:
Is this going to affect how it works when doing a manual format (as opposed to format on save)?

Also, can we put the list of embedded scopes in a constant up at the top of the file so it's easy to find later?